src/providers/DevicesProvider/AudioInputProvider.tsx

```diff
@@ -104,11 +104,20 @@ const AudioInputProvider: React.FC = ({ children }) => {
       }
     }
 
+    const callback = (deviceLabels: DeviceLabels): void => {
+      if (deviceLabels === DeviceLabels.Audio || deviceLabels === DeviceLabels.AudioAndVideo) {
+        initAudioInput();
+      }
+    };
+
+    meetingManager.subscribeToDeviceLabelsChange(callback);
+
     initAudioInput();
```

Comment:
This naming is because I added `this.deviceLabels` to subscribe to. But the underlying logic here is indeed subscribing to `deviceLabelTrigger` change. Since I removed the `this.deviceLabels` right now, will rename it to `*DeviceLabelTriggerChange`.